index.js

```diff
@@ -3,146 +3,240 @@ const debug = require("debug")("signalk:signalk-to-nmea2000");
 const util = require("util");
 const toPgn = require("to-n2k").toPgn;
 const _ = require('lodash')
+const path = require('path')
+const fs = require('fs')
 
 module.exports = function(app) {
   var plugin = {};
   var unsubscribes = [];
-  var timer;
+  var timers = []
+  var conversions = load_conversions(app, plugin)
+  conversions = [].concat.apply([], conversions)
+
+  /*
+    Each conversion can specify the sourceType and outputType.
+
+    Source type can be:
+
+      onDelta - You will get all deltas via app.signalk.on('delta', ...). Please do no use this unless absolutely necessary.
+      
+      bacanjs - The conversion should specify a variable called 'keys' which is an array of the Signal K paths that the convesion needs
```

Comment:
Could this be renamed to e.g. `onValueChange`? As far as I understood, `baconjs` refers to specific implementation, not the source concept.

Also: typo (on line 33, too).

Reply:
+1 to `onValueChange` or similar